src/api-requests/get-industry-events.ts

```diff
@@ -43,6 +43,32 @@ export const industryEventsFields = `
   location
   slug
   start_time
+  registration_url
+  virtual_meetup_url
+  virtual_meetup_url_text
+  related_content {
+    newArticles: new_articles {
+        title: name
+        contentDate: originalPublishDate
```

Comment:
contentDate for articles should be originalPublishDate | published_at, original publish date wont be there for the ones published from CMS. We had few articles in other world, which were migrated to our CMS, only those have original publish date.